ROADMAP.md

```diff
@@ -38,3 +38,12 @@ RPC server configuration setting SessionExpirationTime of `int` type has been
 replaced by SessionLifetime of `Duration` type.
 
 Removal of SessionExpirationTime is scheduled for 0.113.0 release.
+
+## Candidate registration via `registerCandidate` method of native NeoToken contract
+
+The original candidate registration command using the `registerCandidate` method
```

Comment:
The original way of Neo candidate registration via `wallet candidate register` CLI command using `registerCandidate` method...